package.json

```diff
@@ -19,22 +19,28 @@
     "lib"
   ],
   "main": "index.js",
+  "type": "module",
   "engines": {
-    "node": ">=8.6"
+    "node": ">=12"
   },
   "scripts": {
     "test": "mocha && tsc -p ./test/types",
-    "cover": "nyc --reporter=text --reporter=html mocha"
+    "lint": "eslint .",
+    "cover": "nyc --reporter=text --reporter=html mocha",
+    "build": "parcel build index.js --target node",
+    "watch": "parcel watch index.js --target node"
```

Comment:
Added linting script and bundler. 
Should I fix the last linter errors and add the linting as part of the tests?